config/deploy.rb

```diff
@@ -45,7 +44,7 @@
 # The .env and config/database.yml are private files that must be
 # manually placed on the deployment system into the shared path.
 # The shared/.env file contains deployment-specific ENV values,
-# and the config/deploy/{APP_ENV}.rb file contains a :default_env.
+# and the config/deploy/server_settings.rb file contains a :default_env.
 set :linked_files, %w(.env config/database.yml)
```

Comment:
When we get everything in pace for puppet (see https://github.com/sul-dlss/puppet/pull/2024), we will want to add additional linked files for the environment configs.  I _think_ these additional file links will need to go into the `config/deploy/{stage}.rb` files so they point to `config/environment/{ENV['APP_ENV'].rb` files.